### PR TITLE
Update/packages/pagination #468

### PR DESCRIPTION
**Pull request by rebecatalkdesk**
Updated Pagination to prepare to add styles: wa-468
- Added `browserlist` to `package.json`
- Removed unneeded classes
- Added `-primary` to root component classes
- Removing the previous and next page buttons depending on the current page and pages length, instead of adding an invisible class
- Changed mobile page display from `h3` to `span` because the font size was huge
- Incremented `quaternary` role to light theme to use on Pagination

Adding Pagination styles: wa-468 
- Created component color-scheme for themes 
- Removed old scss and created new one for the new styling

Updated Pagination story: wa-468
- Added pagination for each theme
- Created empty mock because of an error

Updated Pagination tests: wa-468
- Updated snapshot because of updated html structure
- Updated tests to reflect new component structure

Updated component styles and amount of pages visible: wa-468 
- Updated quaternary color to reflect updated color in ui kit
- Changed the amount of pages visible around current page from 4 to 2
- Reverted change that would remove the prev and next buttons if the user was in the first or last page and added visibility collapse to make the button disappear, preventing the paginator from skipping around
- Added container fluid div because it was missing and messing with the display
- Added col classes for alignment in various breakpoints
- Added margin auto to prev and next button to prevent the buttons moving around
- Updated tests to reflect changes

### CODE REVIEW

**Review by LuisConceicaoTD — APPROVED** [review]

**Review by Mar0cas — CHANGES_REQUESTED** [review]
Please review the pagination flickering when changing pages.
Review with design the state of the `prev` & `next` button, if they are removed from the DOM or if they go to `disabled` state.

File: packages/molecules/pagination/src/Pagination.tsx [hunk 1]
```diff
@@ -20,30 +19,31 @@ const Pagination = ({
     >
       <div className="row">
         <div className="col-14 pagination__content">
```
Comment (Mar0cas) [inline packages/molecules/pagination/src/Pagination.tsx]: Remove `col-14` use `col-12`

File: packages/molecules/pagination/src/styles/Pagination.scss [hunk 2]
```diff
@@ -1,9 +1,61 @@
 @import 'config';
+@import 'color-schemes/index';
+@include theme-generator($pagination-ui-kit-color-schemes);
+
 
 .section__pagination {
```
Comment (Mar0cas) [inline packages/molecules/pagination/src/styles/Pagination.scss:6]: In this file, find where we are using margin auto to push prev & next buttons to the outside, and align all elements to the middle.

File: packages/molecules/pagination/src/Pagination.tsx [hunk 3]
```diff
@@ -10,40 +10,41 @@ const Pagination = ({
...
       className={Utils.visuals(className as string)}
       data-cards={total}
       data-pages={pages.length}
     >
-      <div className="row">
-        <div className="col-14 pagination__content">
-          {current === 1 ||
+      <div className={'container-fluid'}>
+        <div className="row">
+          <div className="col-10 offset-1 col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-lg-4 offset-lg-4 pagination__content">
```
Comment (Mar0cas) [inline packages/molecules/pagination/src/Pagination.tsx:23]: Switch all bootstrap classes to `col-12`.

**Review by Mar0cas — CHANGES_REQUESTED** [review]

**Review by Mar0cas — APPROVED** [review]

**Pull request merged**